app/mailers/application_mailer.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'test@mailer.com' if Rails.env.test? || Rails.env.development?
```

Comment:
To prevent these feature flags, like Rails.env.something, to appear, we use test/development/production.rb files. Please correct me if I'm wrong but you can configure default from somewhere there.